mmv1/third_party/terraform/tests/resource_dns_response_policy_rule_test.go.erb

```diff
@@ -39,6 +39,73 @@ func TestAccDNSResponsePolicyRule_update(t *testing.T) {
 	})
 }
 
+func TestAccDNSResponsePolicyRuleBehavior_update(t *testing.T) {
+	t.Parallel()
+
+	responsePolicyRuleSuffix := randString(t, 10)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProvidersOiCS,
+		CheckDestroy: testAccCheckDNSResponsePolicyRuleDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDnsResponsePolicyRuleBehavior_privateUpdate(responsePolicyRuleSuffix, "network-1"),
+			},
+			{
+				ResourceName:      "google_dns_response_policy_rule.example-response-policy-rule",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccDnsResponsePolicyRuleBehavior_privateUpdate(responsePolicyRuleSuffix, "network-2"),
```

Comment:
Instead of testing the updatability of `network_url` in `google_dns_response_policy`, can we use this to update the `behavior` field?

Reply:
Corrected test to focus on the usage of behavior property now.  note that this property has only 2 valid states, and a conflicts-with type constraint with localData element